### PR TITLE
use strict=true in docs

### PR DESCRIPTION
**Pull request by dpo**

### CODE REVIEW

**Comment by dpo** [comment]
We need https://github.com/JuliaSmoothOptimizers/BenchmarkProfiles.jl/pull/84 and a new release of BenchmarkProfiles first.

**Comment by tmigot** [comment]
> We need [JuliaSmoothOptimizers/BenchmarkProfiles.jl#84](https://github.com/JuliaSmoothOptimizers/BenchmarkProfiles.jl/pull/84) and a new release of BenchmarkProfiles first.

It has been merged, thanks. We now have a new release for BenchmarkProfiles ( 0.4.1 )

File: .github/workflows/Documentation.yml [hunk 1]
```diff
@@ -16,9 +16,16 @@ jobs:
           version: '1'
       - name: Install dependencies
         run: julia --project=docs -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
+      - name: Setup tmate session
+        uses: mxschmitt/action-tmate@v3
```
Comment (dpo) [inline .github/workflows/Documentation.yml:20]: This should be removed before merging.

**Comment by dpo** [comment]
These docs do not *want* to be built. I'll have to debug later.

**Comment by dpo** [comment]
At long last, the docs build!!!

**Review by tmigot — APPROVED** [review]
This one was epic! Great job, thanks.

**Pull request merged**